src/commands/bootstrap/index.ts

```diff
@@ -105,6 +115,315 @@ function writeEnvFile({
   writeFileSync(fileName, content, 'utf8');
 }
 
+async function createBranch({
+  projectId,
+  apiClient,
+  name,
+}: {
+  appName: string;
+  projectId: string;
+  apiClient: Api<unknown>;
+  name: string;
+}) {
+  const {
+    data: { branch },
+  } = await retryOnLock(() =>
+    apiClient.createProjectBranch(projectId, {
+      branch: {
+        name,
+      },
+      endpoints: [
+        {
+          type: EndpointType.ReadWrite,
+        },
+      ],
+    }),
+  );
+
+  return branch;
+}
+
+async function createDatabase({
+  appName,
+  projectId,
+  branchId,
+  apiClient,
+  ownerRole,
+}: {
+  appName: string;
+  projectId: string;
+  branchId: string;
+  apiClient: Api<unknown>;
+  ownerRole?: string;
+}): Promise<Database> {
+  const {
+    data: { database },
+  } = await retryOnLock(() =>
+    apiClient.createProjectBranchDatabase(projectId, branchId, {
+      database: {
+        name: `${appName}-${cryptoRandomString({
+          length: 5,
+          type: 'url-safe',
+        })}-db`,
+        owner_name: ownerRole || DEFAULT_NEON_ROLE_NAME,
+      },
+    }),
+  );
+
+  return database;
+}
+
+function applyMigrations({
+  options,
+  appName,
+  connectionString,
+}: {
+  options: BootstrapOptions;
+  appName: string;
+  connectionString?: string;
+}) {
+  try {
+    let envVar = '';
+    if (connectionString) {
+      envVar = `DATABASE_URL=${connectionString}`;
+    }
+
+    execSync(`${envVar} ${options.packageManager} run db:migrate`, {
```

Comment:
you can pass env var directly as an option

Reply:
otherwise connection string with a quote would probably break the command

Reply:
Ah, great idea! I'm just passing it in `env` now.